src/toolbars/MeterToolBar.cpp

```diff
@@ -187,7 +188,7 @@ void MeterToolBar::Populate()
    if( mWhichMeters & kWithPlayMeter ){
       mPlaySetupButton = safenew AButton(this);
       mPlaySetupButton->SetLabel({});
-      mPlaySetupButton->SetName(_("Playback"));
+      mPlaySetupButton->SetName(wxString::Format(_("Playback meter peak %d db"), -DecibelScaleCutoff.GetDefault()));
```

Comment:
Did you confirm that the buttons reformat after the value is changed and the Preferences dialog is dismissed?

If that doesn't work, then I maybe commit 3598fe4 added enough calls to Invalidate().